Dart Challange/unit_test_1/lib/unit_test_function.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:io';
+
+List<int> getListDivisors(int number) {
+  if (number == 0) {
+    stdout.write("Vô số ước số \n");
+    return [];
+  }
+  List<int> _listDivisors = [1];
```

Comment:
flutter_lints shows the error. Please check [document](https://dart.dev/tools/linter-rules/no_leading_underscores_for_local_identifiers) and fix it